rootToH5converter/Jet.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, eta, phi, pt, mass, flavor, n_charged=None, n_neutral=None, c
         self.chargedHadronEnergyFraction = ch_hef
         self.neutralHadronEnergyFraction = ne_hef
     
-        if self.chargedHadronEnergyFraction is None:
+        if self.chargedHadronEnergyFraction is None and self.n_charged is not None and self.n_neutral is not None:
```

Comment:
Do we handle properly the case when chargedHadronEnergyFraction can't be found/calculated?

Reply:
I didn't check. It should stay `None` after the function, but I didn't check what the values are in the h5. Is there a place place where `None` are converted to NaN at a later stage?
The best would be to add branches to the PFnanoAOD before the converter.

Reply:
It's `nan` in the h5 file (not some random numbers), maybe it's good enough for now?

Reply:
Yes, I think it's good.